evaluate_tflite.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function
+
+import argparse
+import numpy as np
+import wave
+import csv
+import sys
+
+from six.moves import zip, range
+from multiprocessing import JoinableQueue, Pool, Process, Queue, cpu_count
+from deepspeech import Model
+
+from util.text import levenshtein
+from evaluate_tools import process_decode_result, calculate_report
+
+r'''
+This module should e self-contained:
+  - build a libdeepspeech.so with TFLite:
+    - add a dep in native_client/BUILD against TFlite: '//tensorflow:linux_x86_64': [ "//tensorflow/contrib/lite/kernels:builtin_ops" ]
+    - bazel build [...] --copt=-DUSE_TFLITE [...] //native_client:libdeepspeech.so
```

Comment:
In the future, maybe we should make it a separate target, like `libdeepspeech_tflite.so`, to streamline this process.

Reply:
Yeah, we could look into that.